src/main/resources/db/migration/V1_26__cascade_delete.sql

```diff
@@ -0,0 +1,48 @@
+alter table location
```

Comment:
I've added the cascade to delete a single object but will only delete if no children below.